manifests/instance.pp

```diff
@@ -167,14 +182,14 @@
         high_port => $port,
         seltype   => 'ldap_port_t',
         protocol  => 'tcp',
-        before => [
-          Service["dirsrv@${title}"],
+        before    => [
```

Comment:
If the instance is disabled, should this `selinux_port` resource on the unsecure port be `ensure=>absent`?